DataFormats/EgammaTrackReco/src/classes_def.xml

```diff
@@ -7,8 +7,8 @@
    <class name="edm::ValueMap<reco::CaloClusterPtrVector>" />
    <class name="edm::Wrapper<edm::ValueMap<reco::CaloClusterPtr> >" />
    <class name="edm::Wrapper<edm::ValueMap<reco::CaloClusterPtrVector> >" />
-   <class name="reco::ConversionTrack" ClassVersion="10">
-    <version ClassVersion="10" checksum="2951396223"/>
+   <class name="reco::ConversionTrack" ClassVersion="11">
+    <version ClassVersion="11" checksum="1446912891"/>
```

Comment:
Add back the old version, together with the new one

Reply:
I do not get what you want. I updated the ConevrsioTrack and if I do not update the version it does not compile

Reply:
```
   <class name="reco::ConversionTrack" ClassVersion="11">
    <version ClassVersion="10" checksum="2951396223"/>
    <version ClassVersion="11" checksum="1446912891"/>
    <field name="traj_" transient="true" />
   </class>
```